.github/workflows/ci.yml

```
@@ -22,18 +22,31 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-name: Continuous Integration
+name: CI
 
 permissions:
   contents: read
 
+.trigger_paths: &trigger_paths
```

actionlint (1.7.11)

[error] unexpected key ".trigger_paths" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)